corpus/impulse/views.py

```diff
@@ -405,15 +405,25 @@ def announcements_management(request):
                     )
                 elif announcement.announcement_type == "NI":
                     # all users who have not registered for impulse
-                    email_ids = list(
-                        User.objects.exclude(
-                            email__in=ImpulseUser.objects.values_list(
-                                "user__email", flat=True
-                            )
-                        ).values_list("email", flat=True)
+                    users = User.objects.exclude(
+                        email__in=ImpulseUser.objects.values_list("user__email", flat=True)
+                    )
+
+                    users = users.exclude(
+                        email__in=[
+                            "impulse_admin",
+                            "embedathon_admin",
+                        ]
                     )
+                    users = users.exclude(is_staff=True)
+                    users = users.exclude(is_superuser=True)
+                    
+                    email_ids = list(users.values_list("email", flat=True))
+                    
                     
             if email_ids is not None:
+                # THIS REPLACES \n WITH <br> IN THE EMAIL CONTENT
+                announcement.content = announcement.content.replace("\n", "<br>")
```

Comment:
Don't use this, use `linebreaks` in the template.

https://stackoverflow.com/a/10270934

Reply:
Done.